docker/Dockerfile

```diff
@@ -24,14 +22,19 @@ RUN bower update --allow-root
 WORKDIR /usr/src/app
 
 # install nano for editing
-RUN apt-get -y install nano vim
+RUN apt-get -y install nano
```

Comment:
why remove vim? :(
I think that we should either not have any editors to keep the image size small. People can just install their favorite editor if they open a shell. Or we should also install vi/vim, which is the standard editor in *nix-like systems

Reply:
This is unresolved